### PR TITLE
Rebalances Windows

### PR DESCRIPTION
**Pull request by CRITAWAKETS**
## About The Pull Request

This PR rebalances windows to be less strong.

## Why It's Good For The Game

Reinforced plasmaglass and plastitanium windows right now have more effective melee HP than a megafauna. By almost double.

Maybe that should be changed.

## Changelog
:cl:
balance: We ran out of the [REDACTED] used to coat our windows, so now they are not basically invincible anymore.
/:cl:

### CODE REVIEW

**Comment by Superyodeler** [comment]
Reverts cringe
Also, modularize pls

**Comment by SynnGraffkin** [comment]
I mean, can't really modularize removals in a file.

**Comment by Superyodeler** [comment]
You comment them out

**Comment by dinnerwoerror** [comment]
MoDuLaRiZe

File: code/game/objects/structures/window.dm [hunk 1]
```diff
@@ -376,18 +376,9 @@
 /obj/structure/window/reinforced/attackby(obj/item/I, mob/living/user, params)
 	switch(state)
 		if(RWINDOW_SECURE)
```
[the remark this answers is not in the document]
Comment (CRITAWAKETS) [reply code/game/objects/structures/window.dm:378]: the welding step is removed though

[the remark this answers is not in the document]
Comment (MarkSuckerberg) [reply code/game/objects/structures/window.dm:378]: Exactly, that's why there's the
```
/* block comment
    reverted code
bock comment end */

[the remark this answers is not in the document]
Comment (CRITAWAKETS) [reply code/game/objects/structures/window.dm:378]: oh im dumb

**Comment by dee-dubya** [comment]
The only major requirements I have for r-windows are as follows:
- beating it with a regular toolbox should take more time than disassembly would
- disassembly time should remain as-is, they're already the weakest component of any reinforced area and does not need to be made easier to break into

**Comment by DerFlammenwerfer** [comment]
If you can nerf normal reinforced windows without nerfing the reinforced Plasmaglass and plastitanium glass windows, fine, but as it stands, nerfing them across the board is going to make people never build them even more.

**Pull request closed without merging**